components/org.wso2.identity.webhook.common.event.handler/src/main/java/org/wso2/identity/webhook/common/event/handler/api/builder/TokensEventPayloadBuilder.java

```diff
@@ -0,0 +1,21 @@
+package org.wso2.identity.webhook.common.event.handler.api.builder;
+
+import org.wso2.carbon.identity.event.IdentityEventException;
+import org.wso2.carbon.identity.event.publisher.api.model.EventPayload;
+import org.wso2.identity.webhook.common.event.handler.api.constants.Constants;
+import org.wso2.identity.webhook.common.event.handler.api.model.EventData;
+
+public interface TokensEventPayloadBuilder {
```

Comment:
Make it TokenEventPayloadBuilder

Reply:
Fixed 9df996d3826520879c0a51df744f498263255025